packages/components/src/internal/query/api.ts

```diff
@@ -994,6 +994,39 @@ export function saveRows(options: Query.SaveRowsOptions): Promise<Query.SaveRows
     });
 }
 
+function splitRowsByContainer(rows: any[], containerField: string): Record<string, any[]> {
+    const containerRows = {};
+    rows.forEach(row => {
+        const container = caseInsensitive(row, containerField);
+        if (!containerRows[container]) containerRows[container] = [];
+        containerRows[container].push(row);
+    });
+
+    return containerRows;
+}
+
+export function saveRowsByContainer(options: Query.SaveRowsOptions, containerField: string = 'ContainerPath'): void {
+    const commands = [];
```

Comment:
This should be typed as 

```ts
const commands: Query.Command[] = [];
```

Doing to exposes the fact that `containerPath` is not configured as part of the interface in `@labkey/api`.